src/components/Card/Card.mdx

```diff
@@ -208,6 +209,14 @@ This example can be found in the [Turbine homepage](https://nypl-ds-test-app.ver
 
 <Canvas of={CardStories.CardFullClickTurbineExample} />
 
+## Card With Full-Click Functionality and Tooltip
+
+The `Tooltip` relies on pointer events that are overridden by the card's `LinkOverlay`.
+
+If you need a tooltip within a clickable `Card`, wrap the entire `Card` component inside the `Tooltip` and use the `placement` and `offset` props to move the `Tooltip` to display next to the intended reference. Keep in mind that `offset` is static pixel values, so check where the `Tooltip` displays on all breakpoints.
```

Comment:
I'm a bit inclined to say add an example of the responsive `offset` values since you mention it, but I think it may be enough to say for now to keep track of it through state and maybe the `useNYPLBreakpoint` hook, if the app needs to update the value.